admin_manual/ai/app_translate.rst

```diff
@@ -94,7 +94,7 @@ Positive:
 * the trained model is freely available, and thus can be run on-premises
 * the training data is freely available, making it possible to check or correct for bias or optimise the performance and CO2 usage.
 
-Learn more about the Nextcloud Ethical AI Rating `in our blog<https://nextcloud.com/blog/nextcloud-ethical-ai-rating/>`.
+Learn more about the Nextcloud Ethical AI Rating `in our blog <https://nextcloud.com/blog/nextcloud-ethical-ai-rating/>`.
```

Comment:
Link is broken:
![grafik](https://github.com/user-attachments/assets/be87c6fc-283a-4f9b-9078-cdfc095502d4)


```suggestion
Learn more about the Nextcloud Ethical AI Rating `in our blog <https://nextcloud.com/blog/nextcloud-ethical-ai-rating/>_`.
```